clients/google-api-services-compute/v1/1.30.1/com/google/api/services/compute/model/AttachedDiskInitializeParams.java

```diff
@@ -51,7 +51,7 @@
   private java.lang.String diskName;
 
   /**
-   * Specifies the size of the disk in base-2 GB. The size must be at least 10 GB. If you specify a
+   * Specifies the size of the disk in GB. The size must be at least 10 GB. If you specify a
```

Comment:
Is the intent here to distinguish between gigabytes and gibibytes?

Reply:
Yes.

Reply:
Then probably wroth spelling this out. Is there any normative documentation of this somewhere?

Reply:
Base-2 doesn't make sense at all and is certainly wrong. I guess it may have been suggesting GiB rather than GB? @jayantjain93 can you confirm?